app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -95,6 +95,7 @@ class EventsFragment : Fragment() {
         showNoInternetScreen(isNetworkConnected())
 
         rootView.retry.setOnClickListener {
+            eventsViewModel.loadLocationEvents(eventsViewModel.savedLocation.toString())
```

Comment:
This will load events even when there is no internet connection right ?

Reply:
Yup, it tries to load the events, but as there is an error loading events, Toast message pops up.

Reply:
PR Updated. Please check